src/components/attendee/ScheduleOverlay/ScheduleEvents.tsx

```diff
@@ -49,7 +37,7 @@ export const ScheduleEvents: React.FC<ScheduleEventsProps> = ({
   showPersonalisedSchedule,
   selectedDayIndex,
 }) => {
-  const { space } = useWorldAndSpaceByParams();
+  const { world } = useWorldAndSpaceByParams();
```

Comment:
This hook also returns `worldId`. Is the id all that is needed? Might be safer to constrain the destructuring to that simple string then, and maybe `isLoading` and `isLoaded` if some extra checks are required

Reply:
[8f7d0bc](https://github.com/sparkletown/sparkle/pull/2964/commits/8f7d0bcb22aad901b7d6456cb7b36d6787fe4ac1)